### PR TITLE
add alignment score calculator

### PR DESCRIPTION
**Pull request by RonitBarman**

### CODE REVIEW

File: scripts/alignment_score.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
+from re import I
+import sys
+import os
+
+# get indexes from file paths given a delimeter, default space
+def scrapeIndexes(path: str, delim: str = " ")-> list:
```
Comment (wangpatrick57) [inline scripts/alignment_score.py:6]: try to follow the style of the rest of the codebase, which is camel_case and no function annotations

File: scripts/alignment_score.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    indexes = []
+    for line in lines:
+        indexes.append(int(
+            line.strip("\n").split(delim)[0]
+        ))
+    return indexes
+
+# calculate alignment score using needleman-wunsch algorithm given
+# 2 index files, match, mismatch, and gap penalty values
+def alignmentScore(indexA: list, indexB: list, match: int, miss: int, gap: int):
```
Comment (wangpatrick57) [inline scripts/alignment_score.py:18]: the main point is creating an alignment right? not just getting an alignment score. if so maybe this function should be slightly modified?

File: scripts/alignment_score.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+if __name__ == "__main__":
+    # pathA = sys.argv[1]
+    # pathB = sys.argv[2]
+    print(alignmentScore(
+        "A T G C T".split(" "),
+        "A G C T".split(" "),
+        match=1,
+        miss=-1,
+        gap=-2,
+    ))
```
Comment (wangpatrick57) [inline scripts/alignment_score.py:109]: are these values the standard values or ones you put arbitrarily? if they're standard let's make them default values in the function signature

**Review by wangpatrick57 — CHANGES_REQUESTED** [review]
I'll trust you on the algorithm lol, so it's just a few simple things

File: scripts/alignment_score.py (hunk 3 above)
Comment (RonitBarman) [reply scripts/alignment_score.py:109]: There's not really a standard value, I think it depends on what you value more. For example, if you make the mismatch penalty higher than the gap penalty, then the algo will try to minimize the gaps but the number of mismatches will probably be high.

File: scripts/alignment_score.py (hunk 2 above)
Comment (RonitBarman) [reply scripts/alignment_score.py]: the function does both, it appends the score while creating the alignment

Comment (wangpatrick57) [reply scripts/alignment_score.py]: I see, maybe rename the function to "create_alignment"?

File: scripts/alignment_score.py (hunk 3 above)
Comment (wangpatrick57) [reply scripts/alignment_score.py:110]: ahh I see, then I trust you'll be able to experimentally find good values when aligning the graphlet sequences?

File: scripts/alignment_score.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+            rlist.append(
+                line.strip("\n").split(delim)[0] # only get index value
+            )
+    return rlist
+
+def alignment_score(indexA: list, indexB: list, match: int, miss: int, gap: int):
+    matrix = [[0]*(len(indexA) + 1) for i in range(0, len(indexB) + 1)] # empty matrix
+    rowval = [x for x in indexB]
+    rowval.insert(0, -1)
+    colval = [x for x in indexA]
```
Comment (wangpatrick57) [inline scripts/alignment_score.py:23]: super small thing, can you also rename indexA and indexB to index_a and index_b?

**Review by wangpatrick57 — CHANGES_REQUESTED** [review]

File: scripts/alignment_score.py (hunk 4 above)
Comment (RonitBarman) [reply scripts/alignment_score.py:23]: oh yea forgot to do that

File: scripts/alignment_score.py (hunk 3 above)
Comment (RonitBarman) [reply scripts/alignment_score.py:110]: yes, I'll try to

**Review by wangpatrick57 — APPROVED** [review]
lgtm

**Pull request merged**